src/modules/vtol_att_control/standard.cpp

```diff
@@ -154,15 +153,14 @@ void Standard::update_vtol_state()
 			    can_transition_on_ground()) {
 				_vtol_schedule.flight_mode = vtol_mode::MC_MODE;
 			}
-
 		}
 
 	} else {
 		// the transition to fw mode switch is on
 		if (_vtol_schedule.flight_mode == vtol_mode::MC_MODE || _vtol_schedule.flight_mode == vtol_mode::TRANSITION_TO_MC) {
 			// start transition to fw mode
 			/* NOTE: The failsafe transition to fixed-wing was removed because it can result in an
-			 * unsafe flying state. */
+			* unsafe flying state. */
```

Comment:
```suggestion
			 * unsafe flying state. */
```

Reply:
This is a change that came from `make format` with Astyle 2.06. Happy to commit your suggestion though.

Reply:
Thanks for switching it back. I hope that's in line with the format check using the new version otherwise it would have failed earlier.

Good to know. We exclusively use astyle 3.1 for quite some time now. I can recommend to update for consistency and also because the newer version has a bunch of corner cases fixed.

Reply:
Time to update https://dev.px4.io/master/en/contribute/code.html#code-style-formatting then ;) I thought I was using recommended formatting options.

Reply:
Yes, let me update this. Thanks for the feedback 👍

Reply:
https://github.com/PX4/PX4-user_guide/pull/991

Reply:
btw dev guide and user guide were merged and the above link points to the legacy dev guide that remains atm for old links but doesn't get updated anymore. Here's the PX4 guide now (same astyle version though) https://docs.px4.io/master/en/contribute/code.html#code-style-formatting

Reply:
@ThomasRigi Thanks again for the input on docs. FYI the link you originally posted now redirects to the up to date docs version with the correction. See https://px4.slack.com/archives/C581ST17V/p1610513080281500 for details.